src/components/surfitProfile/Navbar.jsx

```diff
@@ -0,0 +1,52 @@
+import logo from 'assets/icon_logo.svg';
+import profilePic from 'assets/mockup_profile.svg';
+import { React } from 'react';
+import styled from 'styled-components';
+import { theme } from 'styles/theme';
+
+const NavContainer = styled.ul`
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  list-style: none;
+  width: 34.3rem;
```

Comment:
```suggestion
  width: 100%;
  padding: 0 1.6rem;
```
이렇게 하는게 더 좋을듯!! 네브바가 화면 가로 넓이 전체 다 차지하고 있어야 하는게 맞으니까요!